vellum/workflows/triage_message/nodes/process_job_node.py

```diff
@@ -0,0 +1,99 @@
+import logging
+from typing import Optional
+from vellum import (
+    ChatMessagePromptBlock,
+    JinjaPromptBlock,
+    PromptParameters,
+)
+from vellum.workflows.nodes import BaseInlinePromptNode
+from .read_message_node import ReadMessageNode
+
+
+logger = logging.getLogger(__name__)
+
+
+def complete_job():
+    """
+    Mark the job as complete by creating a job session with an end time.
+    Use this when you have successfully completed the job's requirements.
+    """
+    pass
+
+
+def defer_job():
+    """
+    Defer the job for later processing. Use this when you need more information
+    or when the job cannot be completed at this time.
+    """
+    pass
+
+
+def get_job_context(job_id: Optional[str] = None) -> str:
```

Comment:
Instead of this tool + node, we need a start job tool which initiates a job session